samza-core/src/test/scala/org/apache/samza/serializers/TestMetricsSnapshotSerde.scala

```diff
@@ -43,6 +39,9 @@ class TestMetricsSnapshotSerde {
     val snapshot = new MetricsSnapshot(header, metrics)
     val serde = new MetricsSnapshotSerde()
     val bytes = serde.toBytes(snapshot)
-    assertTrue(serde.fromBytes(bytes).equals(metrics))
+    val deserialized = serde.fromBytes(bytes)
+    assertTrue(deserialized.getHeader.jobName.equals(snapshot.getHeader.jobName))
+    assertTrue(deserialized.getHeader.jobId.equals(snapshot.getHeader.jobId))
+    assertTrue(deserialized.getMetrics.immutableMetrics.keySet().equals(snapshot.getMetrics.immutableMetrics.keySet()))
```

Comment:
Remove?
This code branch and test need not be updated.

Reply:
Removed.